setup.py

```diff
@@ -59,5 +59,9 @@
     kwargs.update({
         'ext_modules': cythonize('openmc/data/reconstruct.pyx')
     })
+    import numpy as np
```

Comment:
numpy is required anyway so you can move this import to the top of the file with other imports